modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -401,11 +412,16 @@ To filter a range of ports, use a "start-end" range in string format. For exampl
 | `peerIP` defines the IP address to filter flows by.
 Example: `10.10.10.10`.
 
+| `pktDrops`
+| `boolean`
+| `pktDrops`, to filter flows with packet drops
```

Comment:
Should line 417 be, "`pktDrops` filters flows with packet drops"?

Reply:
I think you're right here.